AcadeMeData/migrations/0001_initial.py

```diff
@@ -0,0 +1,125 @@
+# Generated by Django 3.2.8 on 2021-11-22 16:23
+
+from django.conf import settings
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    dependencies = [
+        ('auth', '0012_alter_user_first_name_max_length'),
```

Comment:
'0012_alter_user_first_name_max_length')

Once you are running make migration and you have already exist data in the database, this is automatically created, 

Please delete db.sqlite3 and run the make migration,
Only clean and relevant files are needed,

Reply:
My mistake